ios/Api.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Api.swift
+//  Magnet
+//
+//  Created by Francisco Jordano on 01/11/2016.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+protocol Api {
+  func get(path: String, callback: ApiCallback)
+  func post(path: String, data: NSDictionary, callback: ApiCallback)
+  func put(path: String, data: NSDictionary, callback: ApiCallback)
+  func delete(path: String, data: NSDictionary, callback: ApiCallback)
+  func mount(path: String, api: Api)
+}
```

Comment:
Unfortunately in swift there is no abstract classes, but the protocol is the interface. Then we use the ApiBase as a superclass for the rest of the apis.

Strictly not necessary but like to keep it.